applications/crossbar/src/modules/cb_rates.erl

```diff
@@ -133,6 +150,23 @@ put(Context) ->
 delete(Context, _RateId) ->
     crossbar_doc:delete(Context).
 
+-spec validate_number(ne_binary(), cb_context:context()) -> cb_context:context().
+validate_number(Phonenumber, Context) ->
+    case length(re:replace(Phonenumber, "[^0-9]", "", [global, {return, list}])) >= 7 of 
```

Comment:
James, let me disagree with you. By default reconcilable numbers are only US numbers.
-define(DEFAULT_RECONCILE_REGEX, <<"^\\+?1?\\d{10}$">>).

This means all other directions with fail with this check.
International users usually just change +1 code to its own country. Which means that they also will be able to check rates to national calls but not international 

Please let me know if I'm wrong